notebooks/07_annotations.ipynb

```diff
@@ -624,9 +624,9 @@
  ],
  "metadata": {
   "kernelspec": {
-   "display_name": "scipy-scipy-interactive-dataviz-bokeh",
+   "display_name": "bk-tutorial",
    "language": "python",
-   "name": "conda-env-scipy-scipy-interactive-dataviz-bokeh-py"
+   "name": "python3"
```

Comment:
Not sure about this change, @pavithraes ?

Reply:
This was committed accidentally!

Reply:
I fixed chapter 7 and 8, they should now use the correct metadata again. I'm working on this from a train with a somewhat unpredictable internet connection... 😅